src/bundle/Resources/public/js/OnlineEditor/buttons/ez-btn-styleslist.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import AlloyEditor from 'alloyeditor';
+import EzButtonDropdown from './ez-btn-dropdown';
+
+export default class EzButtonStylesList extends AlloyEditor.ButtonStylesList {
+    /**
+     * Lifecycle. Renders the UI of the list.
+     *
+     * @instance
+     * @memberof ButtonStylesList
+     * @method render
+     * @return {Object} The content which should be rendered.
+     */
+    render() {
```

Comment:
I can't spot any meaning change in this file in comparison to the original. Why this file was overridden?

I think adding:
```js
AlloyEditor.ButtonDropdown = AlloyEditor.EzBtnDropdown = EzBtnDropdown;
```
In the `ez-btn-dropdown.js` should do the trick.

Reply:
@dew326, that's what I did at the beginning, however, I'm not sure overriding whole `ButtonDropdown` is a good idea, because some other elements which work correctly may depend on it.

The single change is here at line 22: 
`            <EzButtonDropdown {...this.props}>
`

So the new `ButtonDropdown` is used **only** in `ButtonStylesList`, not everywhere else.

Reply:
So if the other components work well with the focus why this one is not?

Reply:
Because other components are not extendable like ButtonStylesList and they don't have long lists of items, which may cause problems. So basically, the only dropdown which may cause this issue is this one as there are many items on the list.